doc/040_backup.rst

```diff
@@ -628,6 +628,12 @@ environment variables. The following lists these environment variables:
     AWS_DEFAULT_REGION                  Amazon S3 default region
     AWS_PROFILE                         Amazon credentials profile (alternative to specifying key and region)
     AWS_SHARED_CREDENTIALS_FILE         Location of the AWS CLI shared credentials file (default: ~/.aws/credentials)
+    RESTIC_AWS_ASSUME_ROLE_ARN          Amazon IAM Role ARN to assume using discovered credentials
+    RESTIC_AWS_ASSUME_ROLE_SESSION_NAME Session Name to use with the role assumption
+    RESTIC_AWS_ASSUME_ROLE_EXTERNAL_ID  External ID to use with the role assumption
+    RESTIC_AWS_ASSUME_ROLE_POLICY       Inline Amazion IAM session policy
```

Comment:
I've tried to come up with a description for `RESTIC_AWS_ASSUME_ROLE_POLICY` and `RESTIC_AWS_ASSUME_ROLE_STS_ENDPOINT` which were missing. Please take a look.

Reply:
I think the policy description is acceptable. If someone wants to use it they'll need to understand how STS policies work anyways and look up AWS documentation.